nix-mineral.nix

```diff
@@ -714,7 +714,7 @@ in
     # Main module
 
     (l.mkIf cfg.enable {
-      boot = l.mkDefault {
+      boot = {
         kernel = l.mkDefault {
```

Comment:
IMO, we should also remove `mkDefault` on `kernel` and `sysctl` too.
And generally only use it on leaves values, unless the leaves form a coherent set that should be overridden altogether.

Reply:
Ah, that what was done in last commit. :+1:

Reply:
Yeah
Should I squash?